domains/ajays.json

```diff
@@ -1,11 +1,11 @@
 {
   "description": "want to use it for hosting my portfolio site",
-  "repo": "https://github.com/AjayIsDev/AjayIsDev.github.io",
+  "repo": "https://github.com/Dev-Ajay-sahani/Ajay_site",
   "owner": {
-    "username": "AjayIsDev",
+    "username": "Dev-Ajay-sahani",
```

Comment:
Doesn't seem that you are the original user, can you prove that you are the original user?

Reply:
last time maintainer om.senjalia helped me in merging you can contact with him he is my friend @CuteDog5695

Reply:
i changed my github username thats why i want to update it

Reply:
updated my repo
@CuteDog5695

Reply:
gmail is same bro u can verify through that i am the original user

Reply:
> gmail is same bro u can verify through that i am the original user

Hm, you do have a point. Can you email cutedog5695@gmail.com (aka my email) with this PR so that I can verify its you?

Reply:
> updated my repo @CuteDog5695

Seems fine to me now, I think now its only about the user thing and PR will be merged

Reply:
i can confirm through gmail should i mail u from that email ?

Reply:
Yes, thats why I gave my email. Just email me the link of the PR with the email and I'll merge it